gnosis/safe/api/base_api.py

```diff
@@ -0,0 +1,46 @@
+from abc import ABC
+from typing import Dict, Optional
+from urllib.parse import urljoin
+
+import requests
+
+from gnosis.eth.ethereum_client import EthereumClient, EthereumNetwork
```

Comment:
I thought you didn't like this kind of import?

Reply:
You are right, missed that 👍🏻